src/megaapi_impl.cpp

```diff
@@ -16729,6 +16776,16 @@ void MegaApiImpl::sendPendingRequests()
 
                     client->putua(type, (byte *)value, 1);
                 }
+                else if (type == ATTR_LAST_PSA)
+                {
+                    if (!value || !atoll(value))
```

Comment:
If the PSA's id 0 were a valid one, it could block the app to set that PSA as seen. Not sure if ids start from 1?